### DISCUSSION TITLE
Readability improvement

### DISCUSSION THREAD

**Discussion by cuprum** [ahyatt/ekg#34] [Ideas]
Hi.

My suggestion is to add a blank line after each note for more readibilyty on tag page. Or, to be more precise, after taglines.
I'm using a dark theme and would prefer that the notes were visually separated from each other.

**Comment by ahyatt** [comment]
This is a good idea, I'll try it out, thanks.

**Comment by ahyatt** [comment]
I've pushed a change to do that on the `develop` branch, thank you for the suggestion!
